Marlin/src/module/planner.h

```diff
@@ -104,11 +108,11 @@ typedef struct {
   uint32_t step_event_count;                // The number of step events required to complete this block
 
   #if EXTRUDERS > 1
-    uint8_t active_extruder;                // The extruder to move (if E move)
+    uint8_t active_extruder_b;              // The extruder to move (if E move)
   #endif
 
   #if ENABLED(MIXING_EXTRUDER)
-    uint32_t mix_steps[MIXING_STEPPERS];    // Scaled steps[E_AXIS] for the mixing steppers
+    MIXER_BLOCK_DEFINITION;                 // Normalized color for the mixing steppers
```

Comment:
Is there any chance this could be something other than `mixer_color_t b_color[MIXING_STEPPERS]` ?

Reply:
I had variantes where only the tool index was stored. But then the mix could be altered while the block is already cued or running.

Reply:
I wanted to have **all** mixer internals inside mixer. What information is stored in the block for the mixer is quite uninteresting to other parts of the code.